README.md

```diff
@@ -15,4 +16,22 @@ markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
 
+## Definition of little $o$ 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+## Definition of Big $O$
+$f(n)\in O(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$
+
+## Proof 
+- If we start with $f(n)\in o(g(n))$ by the deffiniton provided we know that for any positive constant $c$ there exists a point called $n_0$
+  beyond which is always $\ f(n) < c \cdot g(n)$
+- If we were to choose another constant $k$ that is greater then 0 but less then or equal to our origninal constant $c$.
```

Comment:
Could we choose a constant greater than $c$?

Reply:
Yes The constant only needs to be a positive.